HISTORY.rst

```diff
@@ -26,6 +26,10 @@ unreleased
 
 * Fixed an issue with enrollement group certificate encoding 
 
+**IoT DPS updates**
+
+* Removed file extension restriction when user attach certificate for enrollment groups creation/update commands,
```

Comment:
this affects individual enrollments too

Reply:
"Removed file extension restriction for attached certificates in individual enrollments and enrollment groups creation/update commands"

Reply:
right, will update thanks!